Dfe.PrepareConversions/Dfe.PrepareConversions/Pages/ProjectList/Index.cshtml

```diff
@@ -2,151 +2,158 @@
-											</span>
-											<span id="@("opening-date-" + index)">@("Opening date: " + project.ProposedAcademyOpeningDate)</span>
-										}
-									</p>
-								</td>
-							</tr>
-							index++;
-						}
-					}
-					</tbody>
-				</table>
+        @{
+            if (Model.ProjectCount == 0)
+            {
+                <div class="govuk-heading-m govuk-!-margin-top-3">There are no matching results.</div>
+                <p class="govuk-body">Improve your results by:</p>
+                <ul class="govuk-list govuk-list--bullet govuk-body">
+                    <li>double-checking your spelling</li>
+                    <li>searching again with another school</li>
+                    <li>removing some filters</li>
+                </ul>
+            }
+            else
+            {
+                <div class="sort-component">
+                    <nav class="govuk-body sort-component__nav" aria-label="sort-by">
+                        <span class="govuk-!-font-weight-bold">Sorted by:</span> Project created date
+                    </nav>
+                </div>
+                <table class="govuk-table">
+                    <caption class="govuk-table__caption govuk-table__caption--m govuk-visually-hidden">Projects (@Model.ProjectCount)</caption>
+                    <thead class="govuk-table__head">
+                        <tr class="govuk-table__row">
+                            <th class="govuk-table__header app-!-width-three-fifths">Application information</th>
+                            <th class="govuk-table__header govuk-!-text-align-right app-!-width-two-fifths prepare-text-align-right">Project status and dates</th>
+                        </tr>
+                    </thead>
+                    <tbody class="govuk-table__body">
+                        @{
+                            var index = 0;
+                            foreach (ProjectListViewModel project in Model.Projects)
+                            {
+                                <tr class="govuk-table__row" data-cy="select-projectlist-filter-row">
+                                    <td class="govuk-table__cell">
+                                        <h2 class="govuk-caption-l govuk-!-margin-bottom-0 govuk-!-margin-top-1">
+                                            <strong>
+                                                <a id="@("school-name-" + index)" class="govuk-link" asp-page="@Links.TaskList.Index.Page" asp-route-id="@project.Id">@project.SchoolName</a>
+                                            </strong>
+                                            <span id="@("urn-" + index)">URN @project.SchoolURN</span>
+                                        </h2>
+                                        <p class="govuk-!-margin-top-3">
+                                            <div data-cy="route">Route: @(string.IsNullOrWhiteSpace(project.ApplicationReceivedDate) ? "Involuntary conversion" : "Voluntary conversion")</div>
```

Comment:
Have we tested updating an involuntary project yet? To check the lack of an application doesn't throw a null ref at any point? (probably outside of this PR)

Reply:
Good question, will check.

Reply:
Updating seems to work. And generating a template also works. I expect parts of the system will not but I don't want to go through each form to make sure it works with an involuntary project. @rdogra-nimble any thoughts on this? Maybe there's a way of automating this check.

Reply:
@jordan-beard @dneed-nimble School App. form is not displaying in Involuntary convers projects yet in Dev. env. 
Updating seems works fine! Generating a template as well. Yes, we can cover in automation :)